src/main/java/com/palantir/weblogger/WebLoggerResource.java

```diff
@@ -55,7 +56,15 @@ public void logContent(@PathParam("eventName") String eventName, String jsonStri
         jsonLog = addTimestamp(jsonLog);
 
         analyticsLogger.info(jsonLog.toString());
+    }
 
+    @POST
+    @Path("batch")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public void logContent(Map<String, String> events) throws ParseException {
```

Comment:
curious why this method is called `logContent`? why not call it something like `logBatch`? or something more related to the path associated.